### PR TITLE
[NOTEPAD] Simplify FileExists function

### PR DESCRIPTION
**Pull request by katahiromz**
## Purpose
Simplify code.
JIRA issue: [CORE-18837](https://jira.reactos.org/browse/CORE-18837)

## Proposed changes

- Simplify `FileExists` helper function by using `GetFileAttributes` function.

### CODE REVIEW

File: base/applications/notepad/dialog.c [hunk 1]
```diff
@@ -220,13 +220,7 @@ static void AlertPrintError(void)
...
 BOOL FileExists(LPCTSTR szFilename)
 {
-    WIN32_FIND_DATA entry;
-    HANDLE hFile;
-
-    hFile = FindFirstFile(szFilename, &entry);
-    FindClose(hFile);
-
-    return (hFile != INVALID_HANDLE_VALUE);
+    return GetFileAttributes(szFilename) != INVALID_FILE_ATTRIBUTES;
```
Comment (learn-more) [inline base/applications/notepad/dialog.c:223]: Does this behave the same with a directory?

Comment (katahiromz) [reply base/applications/notepad/dialog.c:223]: On the case of a directory, the function will `TRUE` in the current PR.

Comment (katahiromz) [reply base/applications/notepad/dialog.c:223]: Samely.

Comment (katahiromz) [reply base/applications/notepad/dialog.c:223]: But there are some exceptions. `FindFirstFile` function accepts wildcard patterns.

**Review by learn-more — APPROVED** [review]

**Pull request merged**